src/test/app/XChain_test.cpp

```diff
@@ -0,0 +1,5418 @@
+                // std::cout << to_string(attn_json) << '\n';
+                env.tx(attn_json);
+                spendFee(signer);
+                claims.erase(claims.begin(), claims.begin() + cnt);
+            }
+        }
+
+        void
+        sendCreateAttestations(
+            size_t signer_idx,
+            BridgeID bridge,
+            CreateClaimVec& claims)
+        {
+            size_t cnt = (claims.size() > 8) ? 8 : claims.size();
+            STXChainBridge const stBridge(bridge->jvb);
+            AttestationBatch::AttestationClaim* nullc = nullptr;
+            while (!claims.empty())
+            {
+                STXChainAttestationBatch attn_batch{
+                    stBridge,
+                    nullc,
+                    nullc,
+                    claims.begin(),
+                    claims.begin() + cnt};
+                auto batch = attn_batch.getJson(JsonOptions::none);
+                auto signer = bridge->signers[signer_idx].account;
+                auto attn_json = xchain_add_attestation_batch(signer, batch);
+                // std::cout << to_string(attn_json) << '\n';
+                env.tx(attn_json);
+                spendFee(signer);
+                claims.erase(claims.begin(), claims.begin() + cnt);
+            }
+        }
+
+        void
+        sendAttestations()
+        {
+            for (size_t i = 0; i < signers_attns.size(); ++i)
+            {
+                for (auto& [bridge, claims] : signers_attns[i])
+                {
+                    sendAttestations(i, bridge, claims.xfer_claims);
+                    sendCreateAttestations(i, bridge, claims.create_claims);
+                }
+            }
+        }
+
+        void
+        init(jtx::Account const& acct)
+        {
+            accounts[acct].init(env, acct);
+        }
+
+        void
+        reinit_accounts()
+        {
+            for (auto& a : accounts)
+                init(a.first);
+        }
+
+        void
+        receive(jtx::Account const& acct, STAmount amt, size_t divisor = 1)
+        {
+            if (amt.issue() != xrpIssue())
+                return;
+            auto it = accounts.find(acct);
+            if (it == accounts.end())
+            {
+                accounts[acct].init(env, acct);
+                it = accounts.find(acct);
+            }
+            it->second.expectedDiff +=
+                (divisor == 1
+                     ? amt
+                     : divide(
+                           amt, STAmount(amt.issue(), divisor), amt.issue()));
+        }
+
+        void
+        spend(jtx::Account const& acct, STAmount amt, size_t times = 1)
+        {
+            if (amt.issue() != xrpIssue())
+                return;
+            receive(
+                acct,
+                times == 1
+                    ? -amt
+                    : -multiply(
+                          amt, STAmount(amt.issue(), times), amt.issue()));
```

Comment:
I also have a clang build problem here that is identical to the one noted above.